Changelog.md

```diff
@@ -2,6 +2,12 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**8.15.4
```

Comment:
It should be wrapped with `**`, so it is parsed as bold, see how it looks now: https://github.com/sameersbn/docker-gitlab/blob/master/Changelog.md